common/schemas/repository.rb

```diff
@@ -27,6 +27,7 @@
       "is_slug_auto" => {"type" => "boolean", "default" => true},
 
       "ark_shoulder" => {"type" => "string"},
+      "ark_template" => {"type" => "string", "ifmissing" => "error", "default" => ".zd", "pattern" => "\\A\\.[rsz][de]+k?\\z"},
```

Comment:
Need guidance on this. Adding "ifmissing" causes tests to fail. Rationale is that whenever the field is displayed in the repository form it should be required to have something filled in. Is there another way to achieve this?
Or maybe the better alternative would be to fall back to a default template in the minter if this is empty. Downside of this would be that this would be a bit hidden from the user and could be unexpected.